ts4.1/index.d.ts

```diff
@@ -111,12 +115,35 @@ type NormalizeMulti<T, U extends keyof T, L = LastOf<U>> = L extends U
   ? AppendNS<L, Normalize<T[L]>> | NormalizeMulti<T, Exclude<U, L>>
   : never;
 
+type CustomTypeParameters = {
+  returnNull: boolean;
+  returnEmptyString: boolean;
+};
```

Comment:
Can't we simply rely on `type CustomTypeParameters = Pick<TypeOptions, 'returnNull' | 'returnEmptyString'>;`?

In case we can't, we can make both parameters optional instead of using partial in three different places:
```ts
type CustomTypeParameters = {
  returnNull?: boolean;
  returnEmptyString?: boolean;
};
```

Reply:
We can't make that Pick, because the types for returnNull and returnEmptyString would be `true`, not boolean :(

Applied your suggestion about making the keys optional and remove that Partial @pedrodurek